lago/virt.py

```diff
@@ -780,11 +782,14 @@ def copy_to(self, local_path, remote_path):
 
     def copy_from(self, remote_path, local_path, recursive=True):
         with self._scp() as scp:
-            scp.get(
-                recursive=recursive,
-                remote_path=remote_path,
-                local_path=local_path,
-            )
+            try:
+                scp.get(
+                    recursive=recursive,
+                    remote_path=remote_path,
+                    local_path=local_path,
+                )
+            except SCPException:
+                LOGGER.debug('Remote path %s not found' % remote_path)
```

Comment:
Then it's a change in the behavior and you should change the tests.
Though it might require some discussion to decide what to do... If I'm collecting the vdsm logs after a successful run, and the logs are not there, I'd want it to fail and not skip them silently.

Probably a better option is to add something like --skip-on-missing or similar and use than in case of failure in the runs to collect the logs.